rwengine/src/script/ScriptFunctions.hpp

```diff
@@ -43,13 +71,59 @@ inline bool objectInBounds(GameObject* object,
 	        p.y <= max.y && p.z <= max.z);
 }
 template <class Tobj, class Tvec>
-bool objectInArea(const ScriptArguments& args, Tobj* object, const Tvec& min, const Tvec& max, bool marker)
+bool objectInArea(const ScriptArguments& args, const Tobj& object, const Tvec& min, const Tvec& max, bool marker, bool preconditions = true)
+{
+	if (marker) {
+		auto center = (min+max)/2.f;
+		auto radius = glm::abs(max-min);
+		drawAreaIndicator(args, center, radius);
+	}
+	return preconditions && objectInBounds(object, glm::min(min, max), glm::max(min, max));
+}
```

Comment:
We could probably sort `min` and `max` first so we don't have to do the `abs` for the radius. We can also add a comment to the ordering line to mention that min and max may not be always ordered.

We should probably also rename these to pointA and pointB or something.